src/main/java/Task2/Classes/Car.java

```diff
@@ -0,0 +1,83 @@
+package Task2.Classes;
+
+public class Car implements Alarm, Vehicle {
+    public boolean isAlarmOn;
+    private String brand;
+    private String model;
+    private String colour;
+    private int price;
+    private float speedUp;
+    private float slowDown;
+
+
+    public Car(String brand, String model, String colour,int price,float speedUp, float slowDown){
```

Comment:
Can you use setter for the valuable initialization instead of passing a lot of input params into constructor?

Reply:
Okay, i`ll change it immediately

Reply:
I`ve made a new pull request